lib/message_express/async_publisher.rb

```diff
@@ -4,6 +4,7 @@ module MessageExpress
   class AsyncPublisher
     include MessageExpress::Publisher
     include Sidekiq::Worker
+    sidekiq_options queue: ENV['MESSAGE_EXPRESS_SIDEKIQ_QUEUE'] || 'default'
```

Comment:
Ideally `sidekiq_options` should be passed as a configuration of EventExpress instead of using ENV variables.
So the project using the gem would use:
```ruby
MessageExpress.configure do |c|
  c.sidekiq_options queue: ENV['MESSAGE_EXPRESS_SIDEKIQ_QUEUE']
end
```

Reply:
Great to hear from you :D
This looks cool but do you have any idea how do I specify this configuration value in the job definition?

Reply:
@SimonMiaou have a look at this one
https://github.com/test-IO/message_express/pull/12